src/components/products/ProductCard.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
```

Comment:
💡 I think it would've been much better if we had some few extra controls in storybook just so someone can see how it would look with different inputs, 
e.g: 
- Price
- Ratings
- Product name
- Category

Reply:
If you open the story, and inspect on the right bar, you should see what you are looking for.

Reply:
Update: 

Tried again, but found it missing. Which is odd, because I know I was using it when I pushed.

But anywho, it is resolved now.